src/coreclr/jit/emitarm64.cpp

```diff
@@ -24398,6 +24512,7 @@ BYTE* emitter::emitOutput_InstrSve(BYTE* dst, instrDesc* id)
         // Immediate and pattern to general purpose.
         case IF_SVE_BL_1A: // ............iiii ......pppppddddd -- SVE element count
         case IF_SVE_BM_1A: // ............iiii ......pppppddddd -- SVE inc/dec register by element count
+        case IF_SVE_BO_1A: // ...........Xiiii ......pppppddddd -- SVE saturating inc/dec register by element count
```

Comment:
how are we taking care of 64-bit encoding, where `sf` field at bit 20 needs to be 1?

[SQDECB](https://docsmirror.github.io/A64/2023-06/sqdecb_r_rs.html)

Reply:
Thanks for pointing that out; fixed.